.gitignore

```diff
@@ -16,12 +16,14 @@ app/addons/*
 !app/addons/fauxton
 !app/addons/databases
 !app/addons/documents
+!app/addons/dataimporter
 !app/addons/styletests
 !app/addons/cors
 settings.json*
 i18n.json
 !settings.json.default
 !assets/js/plugins/zeroclipboard/ZeroClipboard.swf
+!assets/js/libs/papaparse.js
```

Comment:
reason: assets is not ignored in this gitignore, so we also don't have to whitelist the file. the reason zeroclipboard is listed here is that *swf files where blacklisted some time ago, but it seems that isn't the case any more